[2023-07-31T18:00:02.615+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-07-31T18:00:02.629+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: userCase2.S5 scheduled__2023-07-31T17:00:00+00:00 [queued]>
[2023-07-31T18:00:02.649+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: userCase2.S5 scheduled__2023-07-31T17:00:00+00:00 [queued]>
[2023-07-31T18:00:02.650+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-31T18:00:02.662+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): S5> on 2023-07-31 17:00:00+00:00
[2023-07-31T18:00:02.673+0000] {standard_task_runner.py:57} INFO - Started process 3199 to run task
[2023-07-31T18:00:02.678+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'userCase2', 'S5', 'scheduled__2023-07-31T17:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/userCase.py', '--cfg-path', '/tmp/tmpcr3slqki']
[2023-07-31T18:00:02.680+0000] {standard_task_runner.py:85} INFO - Job 95: Subtask S5
[2023-07-31T18:00:02.719+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-07-31T18:00:02.775+0000] {task_command.py:410} INFO - Running <TaskInstance: userCase2.S5 scheduled__2023-07-31T17:00:00+00:00 [running]> on host 484e6be077e8
[2023-07-31T18:00:02.791+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-07-31T18:00:02.884+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='userCase2' AIRFLOW_CTX_TASK_ID='S5' AIRFLOW_CTX_EXECUTION_DATE='2023-07-31T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-31T17:00:00+00:00'
[2023-07-31T18:00:02.886+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-31T18:00:02.887+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 10']
[2023-07-31T18:00:02.904+0000] {subprocess.py:86} INFO - Output:
[2023-07-31T18:00:12.924+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-31T18:00:13.044+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=userCase2, task_id=S5, execution_date=20230731T170000, start_date=20230731T180002, end_date=20230731T180013
[2023-07-31T18:00:13.074+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-31T18:00:13.099+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
